beacon-chain/p2p/peers/status.go

```diff
@@ -579,6 +594,88 @@ func (p *Status) HighestEpoch() uint64 {
 	return helpers.SlotToEpoch(highestSlot)
 }
 
+func (p *Status) isfromBadIP(pid peer.ID) bool {
+	p.store.RLock()
+	defer p.store.RUnlock()
+
+	peerData, ok := p.store.PeerData(pid)
+	if !ok {
+		return false
+	}
+	if peerData.Address == nil {
+		return false
+	}
+	ip, err := manet.ToIP(peerData.Address)
+	if err != nil {
+		return true
+	}
+	if val, ok := p.ipTracker[ip.String()]; ok {
+		if val > ColocationLimit {
```

Comment:
```suggestion
		if val >= ColocationLimit {
```
Non-strict inequality, probably?

Reply:
Wouldn't limit imply a strict inequality ?

Reply:
Agree, limit is better represented by the strict inequality.